lib/fluent/plugin/out_mongo.rb

```diff
@@ -172,7 +172,6 @@ def client
     def collect_records(chunk)
       records = []
       chunk.msgpack_each {|time, record|
-        record[@time_key] = Time.at(time || record[@time_key]) if @include_time_key
```

Comment:
Umm, it cannot insert Time object with `inject_values_to_record` for now.
But we should avoid to use deprecated `SetTagKeyMixin`.
Should we add a functionality to return `:time` type for inject plugin helper?